cookbook/deployment/cluster/config_flyte_deploy.rst

```diff
@@ -0,0 +1,181 @@
+
+You can view existing attributes for which tags can be assigned by visiting ``protocol://<host>/api/v1/matchable_attributes?resource_type=3``.
+
+Execution Cluster Label
+=======================
+This allows forcing a matching execution to consistently execute on a specific Kubernetes cluster.
+
+You can set this using flyte-cli:
+
+.. prompt:: bash
+
+   flyte-cli -h localhost:30081 -p flyteexamples -d development update-execution-cluster-label --value mycluster
+
+.. tip::
+   The equivalent command in ``flytectl`` is:
+
+   .. prompt:: bash
+      flytectl update execution-cluster-label
+
+   Refer to the :ref:`docs <flytectl:flytectl_update_execution-cluster-label>` to learn more about the command and its supported flag(s).
+
+*********
+Hierarchy
+*********
+Increasing specificity defines how matchable resource attributes get applied. The available configurations, in order of decreasing specifity, are:
+
+#. Domain, project, workflow name, and launch plan.
+
+#. Domain, project, and workflow name
+
+#. Domain and project
+
+#. Domain
+
+Default values for all and per-domain attributes may be specified in the FlyteAdmin config as documented in the :ref:`config_resource_types`.
+
+Example
+=======
+If the database includes the following:
+
++------------+--------------+----------+-------------+-----------+
+| Domain     | Project      | Workflow | Launch Plan | Tags      |
++============+==============+==========+=============+===========+
+| production | widgetmodels |          |             | critical  |
++------------+--------------+----------+-------------+-----------+
+| production | widgetmodels | Demand   |             | supply    |
++------------+--------------+----------+-------------+-----------+
+
+Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels, Workflow: Demand]** for any launch plan will have a tag value of "supply".
+
+Any inbound ``CreateExecution`` requests with **[Domain: Production, Project: widgetmodels]** for any workflow other than ``Demand`` and any launch plan will have a tag value "critical".
+
+All other inbound CreateExecution requests will use the default values specified in the FlyteAdmin config (if any).
+
+*********
+Debugging
+*********
+Use the `get <https://github.com/flyteorg/flyteidl/blob/ba13965bcfbf7e7bfce40664800aaf1f2a1088a1/protos/flyteidl/service/admin.proto#L395>`__ endpoint
+to see if overrides exist for a specific resource. Hit the URL: ``protocol://<host/api/v1/project_domain_attributes/widgetmodels/production?resource_type=2>``.
+
+To get the global state of the world, list all endpoints. For example, visit ``protocol://<host>/api/v1/matchable_attributes?resource_type=2``.
+
+The resource type enum (int) is defined in the `proto <https://github.com/flyteorg/flyteidl/blob/ba13965bcfbf7e7bfce40664800aaf1f2a1088a1/protos/flyteidl/admin/matchable_resource.proto#L8,L20>`__.
```

Comment:
This can be refrenced from here instead of github link
https://docs.flyte.org/projects/flyteidl/en/docs-revamp/protos/docs/admin/admin.html?highlight=matchable#matchableresource

This is currently in docs-revamp branch but should be checkedin soon
Check this PR for intersphinx links

https://github.com/flyteorg/flytesnacks/pull/223

For reference once the IDL docs reach prod 
Use the following command to see what should be link address  eg:
```

python3 -msphinx.ext.intersphinx /Users/praful/flyteidl/docs/build/html/objects.inv |grep "matchable"
	protos/docs/admin/admin:flyteidl/admin/matchable_resource.proto flyteidl/admin/matchable_resource.proto : protos/docs/admin/admin.html#flyteidl-admin-matchable-resource-proto
	protos/docs/admin/admin:listmatchableattributesrequest ListMatchableAttributesRequest          : protos/docs/admin/admin.html#listmatchableattributesrequest
	protos/docs/admin/admin:listmatchableattributesresponse ListMatchableAttributesResponse         : protos/docs/admin/admin.html#listmatchableattributesresponse
	protos/docs/admin/admin:matchableattributesconfiguration MatchableAttributesConfiguration        : protos/docs/admin/admin.html#matchableattributesconfiguration
	protos/docs/admin/admin:matchableresource MatchableResource                       : protos/docs/admin/admin.html#matchableresource
```

From the above listing

```
protos/docs/admin/admin:matchableresource MatchableResource                       : protos/docs/admin/admin.html#matchableresource
```
Points directly to the MatchableResource